### PR TITLE
add app-server public repo the CF files and rename them

### PR DESCRIPTION
**Pull request by asungur**
- Keep `development` and `production` copies of CF file
- Add public ECR repo of the App Server
- Bug-fix: `cloudformation` instance was only available to `createBaaS` function
- Rename the Log group and move it to CLI app CF file and get it as env var.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# Rest client
+/requests
```
Comment (asungur) [inline .gitignore:107]: For testing via VSCode REST client

File: controllers/instanceController.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
 const aws = require('aws-sdk');
 const fs = require('fs');
 const path = require('path');
+const cloudformation = new aws.CloudFormation();
 
 const createBaaS = (req, res, next) => {
-  const cloudformation = new aws.CloudFormation();
```
Comment (asungur) [inline controllers/instanceController.js:6]: This wasn't available to other functions

File: index.js [hunk 3]
```diff
@@ -22,7 +22,8 @@ const env = (req, res, next) => {
     DBServerIAMRole: process.env.DBServerIAMRole,
     AppServerIAMRoleArn: process.env.AppServerIAMRoleArn,
     DBServerIAMRoleArn: process.env.DBServerIAMRoleArn,
-    ALBListener: process.env.ALBListener
+    ALBListener: process.env.ALBListener,
+    AppServerLG: process.env.AppServerLG,
```
Comment (asungur) [inline index.js:26]: Moved this to CLI CF script and created an env var for it

File: utils/bastion-development.yaml [hunk 4]
```diff
@@ -31,7 +31,7 @@ Resources:
         - FARGATE
       ClusterName: BaaSECSCluster
     DependsOn:
-      - CloudWatchLogsGroup
+      - AppServerLG
```
Comment (asungur) [inline utils/bastion-development.yaml:34]: As discussed, matching naming format with the other log group

File: utils/bastion-development.yaml [hunk 5]
```diff
@@ -83,19 +83,19 @@ Resources:
       ContainerDefinitions:
         - Name: app-server
           Essential: 'true'
-          Image: 'public.ecr.aws/y7d9d7k6/app-server:0.5.0'
+          Image: 'public.ecr.aws/k0c9b1e9/bastion-app-server:0.0.1'
```
Comment (asungur) [inline utils/bastion-development.yaml:86]: This is the new WP `app-server` image

**Review by asungur — COMMENTED** [review]
@ahtrotta @Reillyknut @partemen86 Added the `app-server` image and made some small tweaks

**Review by ahtrotta — APPROVED** [review]
Looks good!

**Pull request merged**